src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -23,6 +23,9 @@
     <Compile Include="$(CommonTestPath)\System\TypeBuilderExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
+      <CodeCoverageAssemblies Include="Microsoft.CSharp" /> 
```

Comment:
I think this needs to be `_CodeCoverageAssemblies` like you had before. Or is this working correctly right now? CodeCoverage.targets has this:

```XML
<_CodeCoverageAssemblies Include="$(CodeCoverageAssemblies)" Condition="'$(CodeCoverageAssemblies)' != ''" />
```

which shouldn't add anything since this is an `ItemGroup` here, not a `PropertyGroup`. It's possible that I don't understand what MSBuild does when you have a Property- and ItemGroup with the same name, though.